tests/testdata/environment.yml

```diff
@@ -2,7 +2,7 @@ name: custom-runtime-environment
 channels:
   - conda-forge
 dependencies:
-  - python == 3.8
+  - python == 3.9
   - scikit-learn == 1.0.2
   - pip:
-      - mlserver == 1.3.0.dev2
+      - git+${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}.git@${GITHUB_REF}
```

Comment:
I switched to installing `mlserver` from git here since worker environment has to match main process for this PR. GitHub Actions should [set these environment variables to the fork/branch](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) so the worker environment installs the same `mlserver`, but default to `SeldonIO/MLServer`'s `master` branch in `tox.ini` otherwise.

Reply:
@lhnwrk is it possible to install from the local mlserver directory so it is easier logic and also locally we might want to be testing from changes done locally?

Reply:
I did go with this option first, but as it turns out this breaks docker build tests since the local mlserver directory is not available inside the container and the template Dockerfile only copies environment file. It has been a hassle to test locally though, what do you think if we use a separate yaml with a pinned version of mlserver for the CLI build test and install local directory for others?